addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -2,6 +2,9 @@
   "name": "<%= htmlComponentName %>",
   "version": "0.0.0",
   "license": "Apache-2.0",
+  "scripts":{
+    "postinstall": "webdriver-manager update"
```

Comment:
This is good to have, but it's slowing down a lot the init time. It would be good to start measuring it and trying to cut it down.

/cc @IgorMinar